commands/polls/polls.js

```diff
@@ -0,0 +1,119 @@
+const { Command } = require('discord.js-commando');
+const { RichEmbed } = require('discord.js');
+const oneLine = require('common-tags').oneLine;
+
+module.exports = class PollCommand extends Command {
+    constructor(client) {
+        super(client, {
+            name: 'poll',
+            group: 'polls',
+            memberName: 'poll',
+            description: 'Creates a poll with up to 10 choices.',
+            examples: ["!poll \"What\'s your favourite food?\" \"Hot Dogs,Pizza,Burgers,Fruits,Veggies\" 10"],
+            args: [
+                {
+                    key: 'question',
+                    prompt: 'What is the poll question?',
+                    type: 'string',
+                    validate: question => {
+                        if (question.length < 101 && question.length > 11) return true;
+                        return 'Polling questions must be between 10 and 100 characters in length.';
+                    }  
+                }, 
+                {
+                    key: 'options',
+                    prompt: 'What options do you want for the poll?',
+                    type: 'string',
+                    validate: options => {
+                        var optionsList = options.split(",");
+                        if (optionsList.length > 1) return true;
+                        return 'Polling options must be greater than one.';
+                    }  
+                },
+                {
+                    key: 'time',
+                    prompt: 'How long should the poll last in minutes?',
```

Comment:
Should this not also be marked optional like in vote?

Reply:
it should yeah

Reply:
fixed